src/index.html

```diff
@@ -4,9 +4,358 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__content">
+        <a href="#" class="logo">
+          <img src="./images/logo.png" alt="moyo logo">
+        </a>
+
+        <nav>
+          <ul class="nav">
+            <li class="nav__item">
+              <a href="/" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                  Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <article class="product-card" data-qa="card">
+          <div class="product-card__img-wrapp">
+            <img
+              class="product-card__img"
+              alt="Apple A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+              src="images/imac.jpeg">
```

Comment:
```suggestion
            <img
              class="product-card__img"
              alt="Apple A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
              src="images/imac.jpeg"
            >
```
Don't forget about formatting